bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -2259,20 +2288,9 @@ public void setBackground(Color color) {
 	device.internal_dispose_GC(hDC, null);
 }
 
-private void setCurrentNativeZoom(int newZoom) {
-	int oldZoom = this.getZoom();
-	if (oldZoom != newZoom) {
-		this.currentNativeZoom = newZoom;
-		// width and height are tied to the current device zoom
-		// they must be reset the the zoom factor changes
-		width = height = -1;
-	}
-}
-
-private int getZoom() {
+private int getCurrentDeviceZoom() {
```

Comment:
Let's revert the renaming back to getZoom()

Reply:
Done